### PR TITLE
Remove scroll on accept

### LINKED ISSUE #88: Cookie Law Update - Scrolling is no longer to be considered an explicit consent
I saw in a similar package that "accept on scroll" is no longer accepted for GDPR.

See: https://github.com/buildo/react-cookie-banner/issues/68

### PR DESCRIPTION
**Pull request by poteirard**
- Remove scroll on accept Fixes #88

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1 @@
 node_modules
-package-lock.json
```
Comment (Mastermindzh) [inline .gitignore:2]: Please remove this from the current PR. Please keep PRs down to a single thing for a better log.

File: README.md [hunk 2]
```diff
@@ -114,47 +107,45 @@ If the decline button is enabled then the (onDecline) prop function can be used,
 
 ## Props
 
-| Prop                     |               Type                | Default value                        | Description                                                                                                                             |
```
Comment (Mastermindzh) [inline README.md:117]: Please revert the layout changes

File: build/index.d.ts [hunk 3]
```diff
@@ -12,7 +12,7 @@ export interface CookieConsentProps {
   children?: React.ReactNode;
   disableStyles?: boolean;
   hideOnAccept?: boolean;
-  onAccept?: ({ acceptedByScrolling }: { acceptedByScrolling?: boolean }) => void;
```
Comment (Mastermindzh) [inline build/index.d.ts:15]: please remove all the build/ changes from the PR, they will be included on release (with a tag).

File: src/index.js [hunk 4]
```diff
@@ -149,23 +113,17 @@ class CookieConsent extends Component {
     const {
       cookieName,
       declineCookieValue,
-      expires,
```
Comment (Mastermindzh) [inline src/index.js:152]: Were these unused?

**Review by Mastermindzh — CHANGES_REQUESTED** [review]
Looks pretty good. Thanks for the update and PR.

- Please remove the package-lock changes (should be a separate PR, I can do it with the build don't worry about it 😄 )
- Remove the build files (pr will be smaller)

Thanks again, don't really keep up with the law that well!

**Comment by Mastermindzh** [comment]
I finally added a contributing guideline, I used the comments above to complement it. Thanks!

https://github.com/Mastermindzh/react-cookie-consent/blob/master/CONTRIBUTING.md

File: src/index.js (hunk 4 above)
Comment (poteirard) [reply src/index.js:152]: Nowhere, I think. That's why I remove it :)

File: README.md (hunk 2 above)
Comment (poteirard) [reply README.md:117]: The layout changed because of the git hook (prettier) since the column now is shorter than before

**Comment by poteirard** [comment]
Addressed the comments. Thanks

File: src/index.js (hunk 4 above)
Comment (Mastermindzh) [reply src/index.js:152]: I'm more worried why they weren't being used tbh, I'll look into them later :smile:

**Review by Mastermindzh — APPROVED** [review]

**Comment by Mastermindzh** [comment]
All approved. I'll merge later today (or tomorrow), gives you some time to do the dependency update as well so we can release in 1 fell swoop :smile:

**Pull request merged**